CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ cmake_dependent_option(
 option(USE_ACL "Use ARM Compute Library" OFF)
 option(USE_ASAN "Use Address Sanitizer" OFF)
 option(USE_CUDA "Use CUDA" ON)
-option(USE_ROCM "Use ROCm" OFF)
+option(USE_ROCM "Use ROCm" ON)
```

Comment:
why are you making Use ROCm ON by default?

Reply:
So that by default we will check whether ROCm is installed in user's system in case it's not installed USE_ROCM will then be [set back to OFF](https://github.com/pytorch/pytorch/blob/f5cf80344fcf7c787710c7dfc2bf1b692fcc371f/cmake/Dependencies.cmake#L750-L752)

Reply:
If we're gonna do this, shouldn't we update the CI script not to explicitly set `USE_ROCM` now?

Reply:
@ezyang setup.py still needs it